common/util.go

```diff
@@ -39,6 +39,15 @@ var (
 	ErrFormatProto = fmt.Errorf("unknown VideoProfile format for protobufs")
 	ErrFormatMime  = fmt.Errorf("unknown VideoProfile format for mime type")
 	ErrFormatExt   = fmt.Errorf("unknown VideoProfile format for extension")
+
+	ext2mime = map[string]string{
```

Comment:
As mentioned earlier, I'd prefer we put this lookup table in LPMS but I know this is more expedient. Letting this through probably means the LPMS changes will never get done though...